src/microbe_stage/Microbe.cs

```diff
@@ -763,6 +763,13 @@ public override void _IntegrateForces(PhysicsDirectBodyState physicsState)
         }
     }
 
+    public override string ToString()
+    {
+        return Species != null! ?
+            $"[{Species.Genus[0]}.{Species.Epithet.Substring(0, 4)}:{GetInstanceId()}]" :
+            string.Empty;
```

Comment:
When null, only return the instance id and for good measure also add the node Name to both strings.

Reply:
Name is too long and of little use.

Reply:
I can see myself comparing the nodes in the remote scene tree with the ones in the game screen while debugging, so this could be of some use.

Reply:
> in the remote scene

It looks like I've missed some important debugging tools. What is that? I'll change InstanceId to that later.

Reply:
When you run Thrive from Godot, in the left panel in the Godot editor you can swap between the local and remote scene trees. When you switch to the remote view you can see all the Nodes the game has created and even edit them a bit in realtime. I've used it myself a few times and it's been pretty handy to see how things actually are in the game to confirm where I've made a mistake.